modules/ROOT/pages/clauses/match.adoc

```diff
@@ -262,151 +228,262 @@ RETURN a, b
 ----
 
-Returns the `ACTED_IN` roles for the movie `Wall Street`.
+.Result
+[role="queryresult",options="header,footer",cols="2*<m"]
+|===
+| movieTitle | director
+| "Wall Street" | "Oliver Stone"
+2+|Rows: 1
+|===
+
+[[where-predicates]]
+== MATCH with WHERE predicates
+
+The `MATCH` clause is often paired with a `WHERE` sub-clause, which adds predicates to refine the patterns, making them more specific.
+These predicates are part of the pattern itself, not just filters applied after matching.
+Thus, the `WHERE` clause should always be placed with its corresponding `MATCH` clause.
+
+.Simple `WHERE` predicate
+[source, cypher]
+----
+MATCH (charlie:Person)-[:ACTED_IN]->(movie:Movie)
+WHERE charlie.name = 'Charlie Sheen'
+RETURN movie.title AS movieTitle
+----
 
 .Result
 [role="queryresult",options="header,footer",cols="1*<m"]
 |===
-| r.role
-| "Gordon Gekko"
-| "Carl Fox"
-| "Bud Fox"
-|Rows: 3
+| movieTitle
+| "Wall Street"
+|Rows: 1
 |===
 
+.More complex `WHERE` predicate
+[source, cypher]
+----
+MATCH (martin:Person)-[:ACTED_IN]->(movie:Movie)
+WHERE martin.name = 'Martin Sheen' AND NOT EXISTS {
+    MATCH (movie)<-[:DIRECTED]-(director:Person {name: 'Oliver Stone'})
+}
+RETURN movie.title AS movieTitle
+----
+
+[NOTE]
+The above query uses an xref:subqueries/existential.adoc[`EXISTS` subquery].
+
+.Result
+[role="queryresult",options="header,footer",cols="1*<m"]
+|===
+| movieTitle
+| "The American President"
+|Rows: 1
+|===
 
-[[relationships-in-depth]]
-== Relationships in depth
+For more information, see the xref:clauses/where.adoc[`WHERE`] page.
+
+[[parameters]]
+== MATCH with parameters
+
+The `MATCH` clause can be used with parameters.
+
+.Parameters
+[source, parameters]
+----
+{
+  "movieTitle": "Wall Street",
+  "actorRole": "Fox"
+}
+----
+
+.Find nodes using paramters
+[source, cypher]
+----
+MATCH (:Movie {title: $movieTitle})<-[r:ACTED_IN]-(p:Person)
+WHERE r.role CONTAINS $actorRole
+RETURN p.name AS actor, r.role AS role
+----
 
 [NOTE]
-====
-Relationships will only be matched once inside a single pattern.
-Read more about this in the section on xref::patterns/reference.adoc#graph-patterns-rules-relationship-uniqueness[relationship uniqueness].
-====
+The above query uses the xref:syntax/operators.adoc#query-operator-comparison-string-specific[`CONTAINS` operator].
+
+.Result
+[role="queryresult",options="header,footer",cols="2*<m"]
+|===
+| actor | role
 
-[[rel-types-with-uncommon-chars]]
-=== Relationship types with uncommon characters
+| "Charlie Sheen" | "Bud Fox"
+| "Martin Sheen"  | "Carl Fox"
 
-Databases occasionally contain relationship types including non-alphanumerical characters, or with spaces in them.
-These are created using backticks (```).
+2+|Rows: 2
+|===
+
+For more information about how to set parameters, see xref:syntax/parameters.adoc[Syntax -> Parameters].
 
-For example, the following query creates a relationship which contains a space (`OLD FRIENDS`) between `Martin Sheen` and `Rob Reiner`.
```

Comment:
the "OLD FRIENDS" example got dropped completely, is this intentional?

Reply:
Yeah - it's pretty niche and seemed better suited the CREATE page if anything.